Makefile

```diff
@@ -108,6 +114,30 @@ tag: ## Set a docker tag to the image.
 push: ## Push docker image.
 	docker push $(IMAGE_PREFIX)pvc-autoresizer:$(IMAGE_TAG)
 
+##@ Chart Testing
+
+.PHONY: ct-lint
+ct-lint: ## Lint and validate a chart.
+	docker run \
+		--rm \
+		--user $(shell id -u $(USER)) \
+		--workdir=/data \
+		--volume $(shell pwd):/data \
+		quay.io/helmpack/chart-testing:v$(CHART_TESTING_VERSION) \
+		ct lint --config ct.yaml
+
+.PHONY: ct-install
+ct-install: ##  Install and test a chart.
+	docker run \
+		--rm \
+		--user $(shell id -u $(USER)) \
+		--network host \
+		--workdir=/data \
+		--volume ~/.kube/config:$(CT_INSTALL_KUBECONFIG):ro \
```

Comment:
maybe needs `--env KUBECONFIG=$(CT_INSTALL_KUBECONFIG)` too?

Reply:
It is correct. Even if specifying the user id, it is unaware of the user name corresponding to it.
I think the mount path can be fixed (for example /tmp/.kubeconfig or so) by setting the environment variable.

Reply:
@pluser @toshipp Sorry, I fixed it.